BasicSwiftSample/SwiftSample/ViewController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ViewController.swift
+//  SwiftSample
+//
+//  Created by Wayne Burkett on 11/7/19.
+//  Copyright © 2019 ForeSee. All rights reserved.
+//
+
+import UIKit
+import ForeSee
+
+class ViewController: UIViewController {
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    ForeSee.setDebugLogEnabled(true)
```

Comment:
Should this be called before `ForeSee.start()`? So that logs during startup can be outputted?